src/mode/terminal.rs

```diff
@@ -170,14 +170,15 @@ where
 
         let numchars = match display_size {
             DisplaySize::Display128x64 => 128,
+            DisplaySize::Display132x64 => 64,
             DisplaySize::Display128x32 => 64,
             DisplaySize::Display96x16 => 24,
         };
 
         // Reset position so we don't end up in some random place of our cleared screen
         let (display_width, display_height) = self.properties.get_size().dimensions();
         self.properties
-            .set_draw_area((0, 0), (display_width, display_height))?;
+            .set_draw_area((6, 32), (display_width, display_height))?;
```

Comment:
These numbers are hacked in. I think I'd add a `display_start` tuple  to DisplaySize and use it anywhere set_draw_area is called to reset to the beginning of the display.